src/features/users/store/store.ts

```diff
@@ -0,0 +1,31 @@
+import { ZetkinUser } from 'utils/types/zetkin';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RemoteItem, remoteItem } from 'utils/storeUtils';
+
+export interface UsersStoreSlice {
+  userItem: RemoteItem<ZetkinUser>;
+}
+
+const initialUsersState: UsersStoreSlice = {
+  userItem: remoteItem(0),
```

Comment:
You are setting the ID of the user here, and then never changing it. Did you consider allowing `userItem` to be `null` when there is no user loaded?

Reply:
I'm not sure how to handle all the errors that comes from allowing the userItem to be null, I can't see a right pattern after looking the other store files :/ so for now I'm changing the remoteItem id so let me know if the userItem still needs to be null

Reply:
Lets talk about it today!